cmd/kubeadm/app/apis/kubeadm/v1alpha2/types.go

```diff
@@ -102,6 +102,9 @@ type MasterConfiguration struct {
 
 	// The cluster name
 	ClusterName string `json:"clusterName,omitempty"`
+
+	// SkipKubeProxyInstall indicates if the kube-proxy will be installed.
+	SkipKubeProxyInstall bool `json:"skipKubeProxyInstall,omitempty"`
```

Comment:
I'm wondering if this should be nested under the KubeProxy struct instead? @neolit123 @liztio thoughts?

Reply:
One who do not wanna use kube-proxy may do not wanna take a deeper look at the `KubeProxy` struct to find what it has... So, I think maybe this place is easer for users who do not wanna install kube-proxy to find what they need.

Reply:
I agree with `KubeProxy.SkipInstall`